iModelJsNodeAddon/presentation/ECPresentationSerializer.h

```diff
@@ -128,4 +131,18 @@ struct DefaultClassSerializer : IECClassSerializer
 	virtual rapidjson::Document _SerializeECClass(ECClassCR, rapidjson::Document::AllocatorType&) override;
 };
 
+/*=================================================================================**//**
+* @bsiclass
++===============+===============+===============+===============+===============+======*/
+struct IModelJsBoundQueryValueSerializer : IBoundQueryValueSerializer
+{
+    ~IModelJsBoundQueryValueSerializer() {}
+    rapidjson::Document _ToJson(BoundQueryECValue const&, rapidjson::Document::AllocatorType*) const;
+    rapidjson::Document _ToJson(BoundQueryId const&, rapidjson::Document::AllocatorType*) const;
+    rapidjson::Document _ToJson(BoundQueryIdSet const&, rapidjson::Document::AllocatorType*) const;
+    rapidjson::Document _ToJson(BoundECValueSet const&, rapidjson::Document::AllocatorType*) const;
+    rapidjson::Document _ToJson(BoundRapidJsonValueSet const&, rapidjson::Document::AllocatorType*) const;
+    std::unique_ptr<BoundQueryValue> _FromJson(BeJsConst const&);

```

Comment:
I think `BeJsConst` docs state it should be passed by value.

Reply:
Removed the reference.